search/src/Components/error/details.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {jsonSchemaTransform} from './utils';
+
+
+const JsonSchemaDetails = (props) => {
+    const {details} = props
+    const tfError = jsonSchemaTransform(details)
+    const tfDisplay = Object.keys(tfError).map((path) => {
+        return (
+            <div>
+                <p>{tfError[path].instances.length} {tfError[path].instances.length > 1 ? "errors": "error"} in the {path}</p>
```

Comment:
I'd be tempted to put these all in accordions. WDYT? I think the different errors need more visual distinction, certainly, either using material Paper for each, or at least making this heading size.

Idiomatic react would use Typography here, rather than p.